tests/Message/SearchRequestTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace Omnipay\Braintree\Message;
+
+use Omnipay\Tests\TestCase;
+
+class SearchRequestTest extends TestCase
```

Comment:
You might wan't to add a test to verify the validation is working as expected since you are implementing a check for the presence of the `purchaseOrderNumber` param.